lib/hexpm/accounts/session.ex

```diff
@@ -1,25 +1,62 @@
 defmodule Hexpm.Accounts.Session do
   use Hexpm.Schema
+  alias Hexpm.Accounts.User
 
   schema "sessions" do
-    field :token, :binary
+    belongs_to :user, User
+    field :uuid, Ecto.UUID, autogenerate: true
```

Comment:
Use UUID for session identifier going forward, also acts as a life raft for the table (i.e., running out of big int PKs).